### PR TITLE
Jmprieur/add ciam dev apps (address PR feedack)

### PR DESCRIPTION
**Pull request by jmprieur**
# Addressing PR feedback for #2399

### CODE REVIEW

File: tests/DevApps/ciam/CiamTest.sln [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿
```
Comment (jennyf19) [inline tests/DevApps/ciam/CiamTest.sln:1]: remove space

**Comment by jennyf19** [comment]
for future reference: would be better to pull from main and start a fresh branch.

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: tests/DevApps/ciam/CiamTest.sln (hunk 1 above)
Comment (jmprieur) [reply tests/DevApps/ciam/CiamTest.sln:1]: false postive. This is the format for sln files.

**Comment by jennyf19** [comment]
https://github.com/AzureAD/microsoft-identity-web/releases/tag/2.13.4

**Pull request merged**